### PR TITLE
196/bug

### PR DESCRIPTION
**Pull request by mglavey**
No longer renders past events in 'Upcoming Events' and 'My Upcoming Events'.

### CODE REVIEW

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 1]
```diff
@@ -23,15 +42,18 @@ export default (props) => {
...
 				<div className="row">
-					{renderEventList(props.myUpcomingEvents)}
+					{sortByDate(renderEventList(props.myUpcomingEvents))}
 				</div>
 			</div>
 		</div>
 	);
 };
 
+
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: You can probably delete this extra newline.

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 2]
```diff
@@ -23,15 +42,18 @@ export default (props) => {
...
 const renderEventList = (events) => {
 	return events.map(event => {
+		if (event.date < today)
+		{
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: I would put this on the line above and indent the whole return statement.

File: src/EventsPage/components/AllUpcomingEvents.js [hunk 3]
```diff
@@ -6,6 +6,22 @@ import _ from 'lodash';
...
+
+if(dd<10) {
+    dd = '0'+dd
+} 
+
+if(mm<10) {
+    mm = '0'+mm
+} 
+
+today = mm + '/' + dd + '/' + yyyy;
```
Comment (vhoof) [inline src/EventsPage/components/AllUpcomingEvents.js]: I think this whole thing could be done with:
```
let today = new Date();
today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
```

Also if we're just using this to cheese the date comparison on line 47, maybe we should move that code into the `renderEventList` function?

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+
+if(dd<10) {
+    dd = '0'+dd
+} 
+
+if(mm<10) {
+    mm = '0'+mm
+} 
+
+today = mm + '/' + dd + '/' + yyyy;
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: Same thing with this re: `toLocaleString`

File: src/EventsPage/components/AllUpcomingEvents.js [hunk 5]
```diff
@@ -6,6 +6,22 @@ import _ from 'lodash';
...
+console.log(today.toString())
```
Comment (vhoof) [inline src/EventsPage/components/AllUpcomingEvents.js]: Can remove this console.log.

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 6]
```diff
@@ -1,6 +1,25 @@
...
+console.log(today.toString())
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: Can remove this console.log.

**Comment by mglavey** [comment]
Added request changes. Tested and is working... should be good to push to qa!

File: src/EventsPage/components/AllUpcomingEvents.js [hunk 7]
```diff
@@ -28,6 +32,7 @@ export default (props) => {
 
 const renderEventList = (events) => {
 	return events.map(event => {
+		if (event.date < today) {
 		return (
```
Comment (vhoof) [inline src/EventsPage/components/AllUpcomingEvents.js]: I think this whole return statement still needs to be indented once to nest under the `if` block.

File: src/EventsPage/components/AllUpcomingEvents.js [hunk 8]
```diff
@@ -6,6 +6,10 @@ import _ from 'lodash';
 
 const sortByDate = (events = []) => (_.orderBy(events, 'date', 'desc'))
 
+let today = new Date();
+today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit'})
```
Comment (vhoof) [inline src/EventsPage/components/AllUpcomingEvents.js]: I'd move these two lines into the `renderEventList` function where we're using them.

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 9]
```diff
@@ -23,25 +42,28 @@ export default (props) => {
...
-						<div className="event-pic"> <img className="img-responsive" src={event.photo} /></div>
-						<div className="event-title"><p>{event.title}</p></div>
-						<div className="event-time"><p>{event.date} <br/> {event.time}</p></div>
-					</div>
-				</Link>
-			</div>
-		);
+	let today = new Date();
+	today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit'})
+		return events.map(event => { if (event.date < today) {
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: I'd move the if clause to the next line and fix the indentation.

File: src/EventsPage/components/MyUpcomingEvents.js [hunk 10]
```diff
@@ -23,25 +42,28 @@ export default (props) => {
...
+				<div key={event.id} className="col-xs-offset-1 col-xs-3">
+					<Link to={`/eventsfeed/${event.id}`}>
+						<div className="event">
+							<div className="event-pic"> <img className="img-responsive" src={event.photo} /></div>
+							<div className="event-title"><p>{event.title}</p></div>
+							<div className="event-time"><p>{event.date} <br/> {event.time}</p></div>
+						</div>
+					</Link>
+				</div>
+			)};
```
Comment (vhoof) [inline src/EventsPage/components/MyUpcomingEvents.js]: This is super-nitpicky, but I'd move the semicolon inside the brace and move the brace to its own line.

File: src/EventsPage/components/MyUpcomingEvents.js (hunk 4 above)
Comment (vhoof) [reply src/EventsPage/components/MyUpcomingEvents.js]: Can delete all this unused code.

File: src/EventsPage/components/AllUpcomingEvents.js [hunk 11]
```diff
@@ -38,6 +43,6 @@ const renderEventList = (events) => {
 					</div>
 				</Link>
 	 		</div>
-		);
+		)};
```
Comment (vhoof) [inline src/EventsPage/components/AllUpcomingEvents.js]: Like the other nitpick, I'd move this brace to its own line (and move the semicolon inside it).

**Pull request merged**